core-api/rest/UCR2-openapi.yaml

```diff
@@ -7800,9 +7800,14 @@ paths:
         progress with the `GET /docks/devices/{dockId}/update/{id}` operation or listen to the WebSocket
         `dock_update_change` event message.
 
-        Response codes:
+        The battery of the remote needs to be at last 20% charged to start the dock firmware update. 
```

Comment:
Typo:

'at last 20%' -> 'at least 20%'

Reply:
Good catch! That's why we have PRs :-)